### PR TITLE
Add jaxb as dependency and allow skipping formatting

### PR DESCRIPTION
**Pull request by galderz**
Fixes extra commits issues in https://github.com/apache/camel-quarkus/pull/1163, and added a commit to allow formatting checks to be skipped altogether (present in quarkus).

### CODE REVIEW

File: extensions-core/core/deployment/pom.xml [hunk 1]
```diff
@@ -49,6 +49,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-arc-deployment</artifactId>
         </dependency>
+        <dependency>
```
Comment (ppalaga) [inline extensions-core/core/deployment/pom.xml:52]:
```suggestion
        <dependency><!-- temporary fix for #1161, remove with Camel 3.3 -->
```

File: extensions-core/core/runtime/pom.xml [hunk 2]
```diff
@@ -56,6 +56,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-arc</artifactId>
         </dependency>
+        <dependency>
```
Comment (ppalaga) [inline extensions-core/core/runtime/pom.xml:59]:
```suggestion
        <dependency><!-- temporary fix for #1161, remove with Camel 3.3 -->
```

**Comment by galderz** [comment]
@ppalaga I'll fix those.

I've verified that format skipping works as expected.

**Comment by galderz** [comment]
@ppalaga Fixed comments.

**Comment by galderz** [comment]
Thx @ppalaga. Anything else missing here? Or can it be integrated?

**Pull request merged**